src/formatPkg.js

```diff
@@ -86,6 +88,7 @@ export default function formatPkg(pkg) {
     homepage: getHomePage(cleaned.homepage, cleaned.repository),
     license,
     keywords,
+    registrySubsets,
```

Comment:
If we only have an array of keywords for now and not actual metadata then let's just call it "computedKeywords".

Reply:
I'm not sure about the name `keywords`

Reply:
technically they are more like identifiers than keywords